src/routes/plans/[id]/+page.svelte

```diff
@@ -414,7 +414,9 @@
   $: if ($initialPlan && browser) {
     // Asynchronously fetch resource types
     effects.getResourceTypes($initialPlan.model_id, data.user).then(initialResourceTypes => {
-      $resourceTypes = initialResourceTypes;
+      externalResourceNames;
+
+      $resourceTypes = [...initialResourceTypes, ...$externalResources.map(({ name, schema }) => ({ name, schema }))];
```

Comment:
Thoughts on making the external resource here unique within their own set? If you upload an external resource for two different simulations you'll get duplicated resource names.

Reply:
Ah true. Do you know if they need to be differentiated in certain cases?

Reply:
I don't think so, it's really the browser and timeline that would make use of these types and they'd only care about the name.

Reply:
Would you mind tweaking the resource type selector in the timeline editor? Looks like it concats the list of external resources onto the list of `resourceTypes` names which results in dupes there.